Hyperqueue-update (#2182)
CSCfi/csc-user-guide: docs/apps/hyperqueue.md

* Puhti: 0.13.0, 0.15.0, 0.16.0
* Mahti: 0.13.0, 0.15.0, 0.16.0
* LUMI: 0.18.0

## Usage

> Initialize recommended version of Hyperqueue on Puhti or Mahti like this:

Reviewer: ```suggestion
Initialize the recommended version of HyperQueue on Puhti and Mahti like this:
```

```bash

> module purge
> module load hyperqueue/<version>

Reviewer: ```suggestion
module load hyperqueue
```

```

Use `module spider` to locate other versions. 

> To access CSC's hyperqueue modules on LUMI,

Reviewer: ```suggestion
To access CSC's HyperQueue modules on LUMI,
```
